packages/storybook/stories/components/modal/documentation.mdx

```diff
@@ -79,6 +79,7 @@ They can be dismissable or not.
 When a **Modal** opening is triggered, it is displayed with a position aligned both horizontally and vertically within the viewport.
 
 **Modal** takes up 100% of the viewport width up to a maximum width of 512px.
+When the screen is a too small, then the **Modal** takes all the available space with a margin of 16px.
```

Comment:
`On small screen, the **Modal** takes all the available space with a margin of 16px.`